crates/ruff_python_formatter/src/expression/parentheses.rs

```diff
@@ -155,7 +155,7 @@ impl<'ast> Format<PyFormatContext<'ast>> for FormatParenthesized<'_, 'ast> {
             } else {
                 group(&format_args![
                     text(self.left),
-                    &line_suffix(&dangling_comments(self.comments)),
+                    &line_suffix(&parenthetical_comments(self.comments)),
```

Comment:
Nit: Should we move the `line_suffix` into `parenthetical_comments` to make it consistent with other comment formatting methods?